src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -523,201 +524,347 @@ public void ActivitySpanIdTests()
         }
 
         /****** WC3 Format tests *****/
+
         [Fact]
-        public void IdFormatTests()
+        public void IdFormat_HierarchicalIsDefault()
         {
-            try
-            {
-                Activity activity;
-
-                // Default format is the default (Hierarchical)
-                activity = new Activity("activity1");
-                activity.Start();
-                Assert.Equal(ActivityIdFormat.Hierarchical, activity.IdFormat);
-                activity.Stop();
+            Activity activity = new Activity("activity1");
+            activity.Start();
+            Assert.Equal(ActivityIdFormat.Hierarchical, activity.IdFormat);
```

Comment:
Activity.Start modifies an AsyncLocal, right?  So should we stop it in a finally to ensure that this doesn't affect other tests?